src/components/Tooltip/Tooltip.tsx

```diff
@@ -8,8 +9,35 @@ export const TooltipProvider = ({
   return <RadixTooltip.Provider {...props}>{children}</RadixTooltip.Provider>;
 };
 
-export const Tooltip = ({ children, ...props }: RadixTooltip.TooltipProps) => {
-  return <RadixTooltip.Root {...props}>{children}</RadixTooltip.Root>;
+export interface TooltipProps extends RadixTooltip.TooltipProps {
+  disabled?: boolean;
+}
+export const Tooltip = ({
+  children,
+  onOpenChange: onOpenChangeProp,
+  open: openProp,
+  disabled = false,
+  ...props
+}: TooltipProps) => {
+  const [open, setOpen] = useState(openProp);
```

Comment:
this will cause inconsistencies with the user state.

we could have `openProp: false` and `open: true` which i think is confusing.I'd say ket's leave the control to the user, in that case we won't need the disabled option. wdyt?